src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14546,6 +14573,7 @@ void VRFCondenserEquipment::VRFOU_CalcCompH(
                                               RoutineName);
     C_cap_operation = this->VRFOU_CapModFactor(
         state, Pipe_h_comp_in, Pipe_h_out_ave, max(min(MinOutdoorUnitPe, RefPHigh), RefPLow), T_suction + this->SH, T_suction + 8, IUMaxCondTemp - 5);
+    C_cap_operation = 1.0;
```

Comment:
This line overwrites the calculation at line 14574, why?

Reply:
It was used for debugging things. I forgot to remove it afterward. Sorry for the confusion.